lib/rubocop-rspec_rails.rb

```diff
@@ -21,7 +21,7 @@
 # https://github.com/rubocop/rubocop-rspec_rails/issues/8
 module RuboCop
   module Cop
-    class AmbiguousCopName # rubocop:disable Style/Documentation
+    class Registry # rubocop:disable Style/Documentation
```

Comment:
Wrong class to override...:bow:
https://github.com/rubocop/rubocop/blob/2e46ac20ed3efb7f2b9ea1168b148034f11e1e46/lib/rubocop/cop/registry.rb#L3-L19

Reply:
🙈 I reviewed that code.

Reply:
By the way, it’s not necessary to reopen the class to prepend a module. E.g. `RuboCop::Cop::Registry.prepend(Module.new`… would work fine.